Python/ceval.c

```diff
@@ -4779,29 +4849,72 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             DISPATCH();
         }
 
+        TARGET(CALL_BUILTIN_FAST_WITH_KEYWORDS) {
```

Comment:
Last I recall, this barely produced any speedups. But that was with the old opcodes and a slower CPython, maybe the percentages have shifted now :).

Reply:
What did you try? Specializing `CALL_NO_KW` or `CALL_KW`? If `CALL_NO_KW` then I wouldn't expect a speedup for `WITH_KEYWORDS`. Now we have a unified `CALL` it handles named arguments as well.

Reply:
Even older -- back in the day of `CALL_FUNCTION_KW` (so `CALL_KW`) I played around with `METH_FASTCALL|METH_KEYWORDS`, it was ~10% faster on microbenchmarks so I abandoned it.

I suspect the gains are even better now. Especially considering it handles methods like you said.